Adyen/UI/UI Style/FormComponentStyle.swift

```diff
@@ -24,6 +24,11 @@ public struct FormComponentStyle: ViewStyle {
     
     /// The footer style.
     public var footer: FormFooterStyle
+
+    /// The helper message style.
+    public var helper: TextStyle = .init(font: .preferredFont(forTextStyle: .body),
```

Comment:
Can we call it simply `Label`, helper is a bit vague.

Reply:
`label` is too generic.
I wan't to avoid a situation where we have another label that looks somehow different but the word `label` is already taken..
How about `hintLabel` ?